src/controllers/requestController.js

```diff
@@ -170,8 +171,9 @@ class RequestController {
         requests = await RequestService.retrieveManagerRequests(directReportIds, { status });
       }
 
-      // eslint-disable-next-line max-len
-      if (isRequester) requests = await RequestService.retrieveAllRequests({ user_id: userData.id, status });
+      if (isRequester) {
```

Comment:
Why can't we include `status` in the block on `line 29`

Reply:
The function on line 174 is different from the code block on line 29 how can that could be to ask who did the upper to include.